practice-one/styles/components/btn.css

```diff
@@ -0,0 +1,10 @@
+.btn{
```

Comment:
check code format `.btn{` to `.btn {`

Reply:
Thank you for your comment. I will fix it